js_machine_front/src/scenes/communityEvents/communityEvents.tsx

```diff
@@ -10,10 +16,29 @@ const sectionStyle = {
   backgroundPosition: 'center',
 };
 
-export const Events: React.FC = memo(() => {
-  return (
-    <div style={ sectionStyle }>
-      <div className="body" />
-    </div>
-  );
-});
+export class Events extends React.PureComponent<{}, EventModel> {
+  constructor(props: {}) {

```

Comment:
no need to create a constructor, you can just create a public field state and set the initial value